common/src/commonMain/kotlin/com/trikot/sample/factories/Bootstrap.kt

```diff
@@ -1,11 +1,11 @@
 package com.trikot.sample.factories
 
-import com.mirego.trikot.kword.I18N
 import com.mirego.trikot.kword.KWord
 
 class Bootstrap {
-    val i18N: I18N = KWord
+    val viewModelControllerFactory: ViewModelControllerFactory = ViewModelControllerFactoryImpl(KWord)
 
-    val viewModelFactory: ViewModelFactory =
-        ViewModelFactoryImpl(this)
+    companion object {
+        val shared = Bootstrap()
```

Comment:
Should this really be publicly available from everywhere?

Reply:
Maybe in a refactor we could change this, right now this is how app access it. I did not wanted to implement the whole injection pattern in a simple sample app.